Update feedback.png (#12970)
MicrosoftDocs/microsoft-365-docs: microsoft-365/security/defender/feedback.md

**Applies to:**
- Microsoft 365 Defender

Your feedback helps us get better at protecting your environment from advanced attacks. Share your experience, impressions, and  requests by providing feedback.

> 1. From the home page of Microsoft Defender, select **Give feedback** and click on **Leave Feedback**. 

Reviewer: ```suggestion
1. From the Microsoft Defender home page, select **Give feedback** and then select **Leave Feedback**. 
```
